docker-entrypoint.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# Set environment passed to container at runtime to global variable ENV.
+set -euo pipefail
+
+sed -i "s/<noscript id=\"env-insertion-point\"><\/noscript>/<script>var ENV=\"${APP_ENV}\"<\/script>/g" /app/react/build/index.html
```

Comment:
Use the .env files instead of this. Setup is already there to read from env files.

Reply:
used env files